crates/wasi-common/yanix/src/sys/linux/filetime.rs

```diff
@@ -6,8 +6,15 @@ use std::fs::File;
 use std::io::Result;
 use std::sync::atomic::{AtomicBool, Ordering::Relaxed};
 
-pub(crate) const UTIME_NOW: i64 = 1_073_741_823;
-pub(crate) const UTIME_OMIT: i64 = 1_073_741_822;
+cfg_if::cfg_if! {
+    if #[cfg(target_pointer_width = "64")] {
+        pub(crate) const UTIME_NOW: i64 = 1_073_741_823;
+        pub(crate) const UTIME_OMIT: i64 = 1_073_741_822;
+    } else {
+        pub(crate) const UTIME_NOW: i32 = 1_073_741_823;
+        pub(crate) const UTIME_OMIT: i32 = 1_073_741_822;
+    }
+}
```

Comment:
We should use `libc::c_long` here. Or even better, can we use `libc::UTIME_NOW` and `libc::UTIME_OMIT` instead of defining our own copies?

Reply:
That's a good point actually. As far as I can remember, the main reason why we'd define them ourselves was to support older `libc` crates where those constants were incorrectly defined for some targets (netbsd and macos are the awkward ones among others). Given that they are now fixed in `libc`, I'm happy to reuse their definitions instead :+1: